packages/sdk/react-shell/src/components/InvitationList/InvitationListItem.tsx

```diff
@@ -63,27 +94,33 @@ export const InvitationListItemImpl = ({
     Invitation.State.CONNECTED,
     Invitation.State.READY_FOR_AUTHENTICATION,
     Invitation.State.AUTHENTICATING,
-  ].includes(status);
+  ].includes(invitationStatus)
+    ? 'pulse'
+    : 'none';
 
-  const avatarError = [Invitation.State.ERROR, Invitation.State.TIMEOUT, Invitation.State.CANCELLED].includes(status);
+  const avatarError = [Invitation.State.ERROR, Invitation.State.TIMEOUT, Invitation.State.CANCELLED].includes(
+    invitationStatus,
+  );
 
   const avatarGreen = [
     Invitation.State.CONNECTED,
     Invitation.State.READY_FOR_AUTHENTICATION,
     Invitation.State.AUTHENTICATING,
     Invitation.State.SUCCESS,
-  ].includes(status);
+  ].includes(invitationStatus);
+
+  const avatarStatus = avatarError ? 'error' : avatarGreen ? 'active' : 'inactive';
 
   return (
-    <ListItem.Root id={invitationCode} classNames='flex gap-2 pli-1 items-center'>
+    <ListItem.Root id={invitationCode} classNames='flex gap-2 pis-3 pie-1 items-center relative'>
       <ListItem.Heading classNames='sr-only'>
         {t('invitation heading') /* todo(thure): Make this more accessible. */}
       </ListItem.Heading>
-      <Avatar.Root
-        animation={avatarAnimation ? 'pulse' : 'none'}
-        status={avatarError ? 'error' : avatarGreen ? 'active' : 'inactive'}
-      >
-        <Avatar.Frame>
+      {authMethod === Invitation.AuthMethod.NONE && (
```

Comment:
I think this one should be based on multi-use rather than auth method, stack is meant to indicate it can be re-used for multiple peers

Reply:
Fixed 👍